src/ParseFile.js

```diff
@@ -317,6 +317,9 @@ const DefaultController = {
     if (XHR) {
       return this.downloadAjax(uri);
     }
+    if (process.env.PARSE_BUILD !== 'node') {
```

Comment:
maybe i don't understand what process.env.PARSE_BUILD is, but the way I'm reading this is:

if we're not in node, complain that we don't have XMLHttpRequest, but XMLHttpRequest is in the browser, not node.  set me straight here :).

Reply:
For browsers that don't support ajax. 

https://www.tutorialspoint.com/ajax/ajax_browser_support.htm

Reply:
PARSE_BUILD is set when building the sdk. 

https://github.com/parse-community/Parse-SDK-JS/blob/master/build_releases.sh